*** array.dtx.org	2006-05-21 19:28:12.000000000 +0200
--- array.dtx	2008-09-06 02:37:03.000000000 +0200
***************
*** 27,32 ****
--- 27,36 ----
  %<+package>\ProvidesPackage{array}
  %<+package>         [2005/08/23 v2.4b Tabular extension package (FMi)]
  %
+ % ***  2008/09/06 Heiko Oberdiek, project `latex-tds':  ***
+ % ***    The description of \arraybackslash is fixed,   ***
+ % ***    see LaTeX bug report `tools/4044'.             ***
+ %
  % \fi
  %
  % \CheckSum{1115}
***************
*** 1939,1946 ****
  %
  % \begin{macro}{\arraybackslash}
  % \changes{v2.4a}{2003/12/17}{(DPC) Macro added (from tabularx)}
! % Restore |\\| for use in array and tabular environment (after
! % |\raggedright| etc.).
  %    \begin{macrocode}
  \def\arraybackslash{\let\\\tabularnewline}
  %    \end{macrocode}
--- 1943,1950 ----
  %
  % \begin{macro}{\arraybackslash}
  % \changes{v2.4a}{2003/12/17}{(DPC) Macro added (from tabularx)}
! % Restore =\\= for use in array and tabular environment (after
! % =\raggedright= etc.).
  %    \begin{macrocode}
  \def\arraybackslash{\let\\\tabularnewline}
  %    \end{macrocode}
