*** athnum.dtx.org	2005-04-01 00:37:00.000000000 +0200
--- athnum.dtx	2006-06-02 19:08:14.000000000 +0200
***************
*** 1,5 ****
--- 1,8 ----
  % \iffalse meta-comment
  %
+ % Patched by Heiko Oberdiek for project latex-tds:
+ % * 2006/06/02: Documentation bug: \CharacterTable without argument removed.
+ %
  % Copyright 1989-2005 Johannes L. Braams and any individual authors
  % listed elsewhere in this file.  All rights reserved.
  % 
***************
*** 27,33 ****
  % extension .ins) which are part of the distribution.
  % \fi
  %% \CheckSum{125}
- %% \CharacterTable
  %\iffalse
  %
  %% This is file `athnum.dtx'
--- 30,35 ----
