*** babel.dtx.org	2005-11-23 22:03:00.000000000 +0100
--- babel.dtx	2006-06-02 19:15:53.000000000 +0200
***************
*** 1,5 ****
--- 1,8 ----
  % \iffalse meta-comment
  %
+ % Patched by Heiko Oberdiek for project latex-tds:
+ % * 2006/06/02: \DocInput{greek.fdd} added.
+ %
  % Copyright 1989-2005 Johannes L. Braams and any individual authors
  % listed elsewhere in this file.  All rights reserved.
  % 
***************
*** 26,32 ****
  % and covered by LPPL is defined by the unpacking scripts (with
  % extension .ins) which are part of the distribution.
  % \fi
! % \CheckSum{3821}
  %%
  % \def\filename{babel.dtx}
  % \let\thisfilename\filename
--- 29,35 ----
  % and covered by LPPL is defined by the unpacking scripts (with
  % extension .ins) which are part of the distribution.
  % \fi
! % \CheckSum{3822}
  %%
  % \def\filename{babel.dtx}
  % \let\thisfilename\filename
***************
*** 5726,5731 ****
--- 5729,5735 ----
  \langdeffile{scottish.dtx}
  %
  \langdeffile{greek.dtx}
+ \DocInput{greek.fdd}
  %
  \langdeffile{frenchb.dtx}
  \langdeffile{italian.dtx}
