*** exscale.dtx.org	2009-09-25 09:56:20.000000000 +0200
--- exscale.dtx	2009-09-25 16:41:35.000000000 +0200
***************
*** 52,58 ****
  %</driver>
  % \fi
  %
! % \CheckSum{39}
  %
  % \changes{v2.1g}{1997/06/16}{Documentation fix.}
  % \changes{v2.1d}{1994/05/09}{Added setting \cs{big}... sizes.}
--- 52,58 ----
  %</driver>
  % \fi
  %
! % \CheckSum{41}
  %
  % \changes{v2.1g}{1997/06/16}{Documentation fix.}
  % \changes{v2.1d}{1994/05/09}{Added setting \cs{big}... sizes.}
