*** finnish.dtx.org	2005-04-01 00:39:00.000000000 +0200
--- finnish.dtx	2006-06-02 19:51:30.000000000 +0200
***************
*** 1,5 ****
--- 1,9 ----
  % \iffalse meta-comment
  %
+ % Patched by Heiko Oberdiek for project latex-tds:
+ % * 2006/06/02: Documentation bug: "=" cannot be used in
+ %               a \changes entry.
+ %
  % Copyright 1989-2005 Johannes L. Braams and any individual authors
  % listed elsewhere in this file.  All rights reserved.
  % 
***************
*** 283,289 ****
  %    \end{macrocode}
  %    then we define two shorthands to be able to specify hyphenation
  %    breakpoints that behave a little different from |\-|.
! % \changes{finnish-1.3p}{2001/11/13}{\texttt{"=} should also use
  %    \cs{bbl@allowhyphens}}
  %    \begin{macrocode}
  \declare@shorthand{finnish}{"-}{\nobreak-\bbl@allowhyphens}
--- 287,293 ----
  %    \end{macrocode}
  %    then we define two shorthands to be able to specify hyphenation
  %    breakpoints that behave a little different from |\-|.
! % \changes{finnish-1.3p}{2001/11/13}{\texttt{\symbol{34}\symbol{61}} should also use
  %    \cs{bbl@allowhyphens}}
  %    \begin{macrocode}
  \declare@shorthand{finnish}{"-}{\nobreak-\bbl@allowhyphens}
