*** hebrew.fdd.org	2011-11-14 04:13:38.000000000 +0100
--- hebrew.fdd	2011-11-14 04:14:41.000000000 +0100
***************
*** 263,269 ****
  %\generateFile{lhecmr.fd}{t}{\from{hebrew.fdd}{LHEcmr,nowarn}}
  %\end{verbatim}
  %
! %  \subsection{The \LHE encoding definition file}
  %
  %    The Hebrew font encoding \LHE{} is based upon the old-code encoding
  %    also known as the Israeli Standard SI-960. Many Hebrew \TeX{}
--- 263,269 ----
  %\generateFile{lhecmr.fd}{t}{\from{hebrew.fdd}{LHEcmr,nowarn}}
  %\end{verbatim}
  %
! %  \subsection{The \LHE{} encoding definition file}
  %
  %    The Hebrew font encoding \LHE{} is based upon the old-code encoding
  %    also known as the Israeli Standard SI-960. Many Hebrew \TeX{}
***************
*** 656,662 ****
  %</LHEredis>
  %    \end{macrocode}
  %
! %  \subsection{The \HEH encoding definition file}
  %
  %    The Hebrew font encoding \HEH{} is based upon an extention by
  %    Microsoft to the ISO-8859-8 standard.  This is an 8bit encoding. The
--- 656,662 ----
  %</LHEredis>
  %    \end{macrocode}
  %
! %  \subsection{The \HEH{} encoding definition file}
  %
  %    The Hebrew font encoding \HEH{} is based upon an extention by
  %    Microsoft to the ISO-8859-8 standard.  This is an 8bit encoding. The
