*** latin.dtx.org	2007-03-19 22:37:51.000000000 +0100
--- latin.dtx	2007-03-19 22:38:38.000000000 +0100
***************
*** 170,175 ****
--- 170,176 ----
  %    \texttt{@} sign, etc.
  %    \begin{macrocode}
  %<*code>
+ \immediate\write16{latin.ldf: Patch for babel/3922 applied.}%
  \LdfInit{latin}{captionslatin}
  %    \end{macrocode}
  %
***************
*** 532,538 ****
  %    \begin{macrocode}
  \addto\extraslatin{\def\LatinMarksOn{\shorthandon{^}\shorthandon{=}}}
  \addto\extraslatin{\def\LatinMarksOff{\shorthandoff{^}\shorthandoff{=}}}
- \addtoextraslatin{\LatinMarksOff}
  %    \end{macrocode}
  %  \end{macro}
  %  \end{macro}
--- 533,538 ----
