*** latin.dtx.org	2008-03-21 23:41:00.000000000 +0100
--- latin.dtx	2008-04-01 21:09:23.000000000 +0200
***************
*** 454,459 ****
--- 454,461 ----
              documentation for explanation}{\@ehd}}}
  \addto\extraslatin{\let\ProsodicMarksOff\relax}
  %    \end{macrocode}
+ %    \end{macro}
+ %    \end{macro}
  %
  %    Then we temporarily set the caret and the equals sign to active
  %    characters so that they can receive their definitions. But first
