*** lppl.tex.org	2009-02-17 12:41:18.000000000 +0100
--- lppl.tex	2011-03-10 16:12:27.000000000 +0100
***************
*** 213,219 ****
        modified version of this component to the user when used
        interactively with that Base Interpreter.
       
!     \item Every component of the Derived Work contains prominent
        notices detailing the nature of the changes to that component,
        or a prominent reference to another file that is distributed as
        part of the Derived Work and that contains a complete and
--- 213,219 ----
        modified version of this component to the user when used
        interactively with that Base Interpreter.
       
!     \item\label{LPPL:item:changelog} Every component of the Derived Work contains prominent
        notices detailing the nature of the changes to that component,
        or a prominent reference to another file that is distributed as
        part of the Derived Work and that contains a complete and
***************
*** 226,232 ****
        of the Derived Work unless those persons have stated explicitly
        that they do provide such support for the Derived Work.
  
!     \item You distribute at least one of the following with the Derived Work:
        \begin{enumerate}
        \item A complete, unmodified copy of the Work; if your
          distribution of a modified component is made by offering
--- 226,232 ----
        of the Derived Work unless those persons have stated explicitly
        that they do provide such support for the Derived Work.
  
!     \item\label{LPPL:item:unmodifiedcopy} You distribute at least one of the following with the Derived Work:
        \begin{enumerate}
        \item A complete, unmodified copy of the Work; if your
          distribution of a modified component is made by offering
***************
*** 482,488 ****
    themselves with the case that a Derived Work is intended to be used
    as a (compatible or incompatible) replacement of the original
    Work. If this is not the case (e.g., if a few lines of code are
!   reused for a completely different task), then clauses 6b and 6d
    shall not apply.
  
    \LPPLsubsubsection{Important Recommendations}
--- 482,489 ----
    themselves with the case that a Derived Work is intended to be used
    as a (compatible or incompatible) replacement of the original
    Work. If this is not the case (e.g., if a few lines of code are
!   reused for a completely different task), then clauses
!   \ref{LPPL:item:changelog} and \ref{LPPL:item:unmodifiedcopy}
    shall not apply.
  
    \LPPLsubsubsection{Important Recommendations}
