*** ltpatch.ltx.org	2009-09-24 17:00:50.000000000 +0200
--- ltpatch.ltx	2011-06-30 00:34:07.000000000 +0200
***************
*** 1,9 ****
  
  %%%%
! %%%% Patch file for the LaTeX2e kernel dated 2009/09/24
! %%%% (2009/09/24)
  
! \def\fmtversion@topatch{2009/09/24} % This patch will not work with
                                      % any other release.
  
  \def\patch@level{0}
--- 1,9 ----
  
  %%%%
! %%%% Patch file for the LaTeX2e kernel dated 2011/06/27
! %%%% (2011/06/27)
  
! \def\fmtversion@topatch{2011/06/27} % This patch will not work with
                                      % any other release.
  
  \def\patch@level{0}
***************
*** 19,25 ****
  *********************************************^^J%
  ltpatch.ltx has fixed certain problems with the `kernel' of LaTeX.^^J%
  Certain other files in the LaTeX distribution have also been updated^^J%
! since the last release (list correct as of 2009/09/24):^^J%
  base/xxxxxxx.dtx..............(patch 1)^^J%
  unpacked/yyyyyyy.cls..........(patch 1)^^J%
  ^^J%
--- 19,25 ----
  *********************************************^^J%
  ltpatch.ltx has fixed certain problems with the `kernel' of LaTeX.^^J%
  Certain other files in the LaTeX distribution have also been updated^^J%
! since the last release (list correct as of 2011/06/27):^^J%
  base/xxxxxxx.dtx..............(patch 1)^^J%
  unpacked/yyyyyyy.cls..........(patch 1)^^J%
  ^^J%
***************
*** 29,38 ****
  \fi
  
  \endinput
- 
- 
- 
- 
- 
- 
- 
--- 29,31 ----
