*** inputenc.dtx.orig	2006-05-05 02:51:46.000000000 +0200
--- inputenc.dtx	2006-05-05 03:04:02.000000000 +0200
***************
*** 2440,2447 ****
  %    \end{macrocode}
  %    But why only for this one encoding? (Answer: because it is a new encoding:
  %    it contains only LICR objects --- Frank)
! % \changes{v1.0g}{2004/05/22}{Changed \cs{textpm} to \cs{pm}
! % \changes{v1.1b}{2006/03/03}{Reverted back to \cs{textpm}}
  %    for consistency.}
  %    \begin{macrocode}
  \DeclareInputText{160}{\nobreakspace}
--- 2440,2447 ----
  %    \end{macrocode}
  %    But why only for this one encoding? (Answer: because it is a new encoding:
  %    it contains only LICR objects --- Frank)
! % \changes{v1.0g}{2004/05/22}{Changed \cs{textpm} to \cs{pm}}
! % \changes{v1.1b}{2006/03/03}{Reverted back to \cs{textpm}
  %    for consistency.}
  %    \begin{macrocode}
  \DeclareInputText{160}{\nobreakspace}
